### PR TITLE
Client change google api

### PR DESCRIPTION
**Pull request by dstrain115**
We should be good to switch the client side to the newest type URL (cirq.google.api).  All downstream processes should now be able to support both type URLs.

This PR switches cirq to use cirq.google.api protos rather than cirq.api.google protos.

### CODE REVIEW

File: cirq/google/api/v2/results_test.py [hunk 1]
```diff
@@ -2,8 +2,8 @@
 import pytest
 
 import cirq
-from cirq.api.google.v2 import result_pb2
 from cirq.google.api import v2
+from cirq.google.api.v2 import result_pb2
```
Comment (maffoo) [inline cirq/google/api/v2/results_test.py]: Alternately, remove this and change to `v2.result_pb2` in the code.

File: cirq/google/api/v2/sweeps_test.py [hunk 2]
```diff
@@ -18,7 +18,7 @@
 import sympy
 
 import cirq
-from cirq.api.google.v2 import run_context_pb2
+from cirq.google.api.v2 import run_context_pb2
```
Comment (maffoo) [inline cirq/google/api/v2/sweeps_test.py]: Could remove this and refer to `v2.run_context_pb2` below.

File: cirq/google/engine/engine.py [hunk 3]
```diff
@@ -575,15 +572,16 @@ def get_job_results(self,
                 parent=job_resource_name))
         result = response['result']
         result_type = result['@type'][len(TYPE_PREFIX):]
-        if result_type == 'cirq.api.google.v1.Result':
-            return self._get_job_results_v1(result)
-        if result_type == 'cirq.api.google.v2.Result':
-            return self._get_job_results_v2(result)
+        print(result_type)
```
Comment (maffoo) [inline cirq/google/engine/engine.py]: Remove print (or use logging).

File: cirq/google/op_serializer.py [hunk 4]
```diff
@@ -20,9 +20,8 @@
 from google.protobuf import json_format
 
 from cirq import devices, ops
-from cirq.api.google import v2
+from cirq.google.api import v2
```
Comment (maffoo) [inline cirq/google/op_serializer.py]: nit: remove blank line.

Comment (dstrain115) [reply cirq/google/op_serializer.py]: Done.

File: cirq/google/engine/engine.py (hunk 3 above)
Comment (dstrain115) [reply cirq/google/engine/engine.py]: Done.

File: cirq/google/api/v2/sweeps_test.py (hunk 2 above)
Comment (dstrain115) [reply cirq/google/api/v2/sweeps_test.py]: Done.

File: cirq/google/api/v2/results_test.py (hunk 1 above)
Comment (dstrain115) [reply cirq/google/api/v2/results_test.py]: Done.

**Pull request merged**